Author: Rémi Duraffort <ivoire@videolan.org>
Description: Install proot into $(DESTDIR)/usr/bin/
Forwarded: not-needed
Last-Update: 2015-01-27

--- a/src/GNUmakefile
+++ b/src/GNUmakefile
@@ -230,13 +230,13 @@
 	-$(RM) -f $(CHECK_OBJECTS) $(CHECK_PROGRAMS) $(CHECK_RESULTS) $(OBJECTS) $(CARE_OBJECTS) $(LOADER_OBJECTS) $(LOADER-m32_OBJECTS) proot care loader/loader loader/loader-m32 cli/care-manual.o $(DEPS) build.h licenses
 
 install: proot
-	$($(quiet)INSTALL) -D $< $(DESTDIR)/$<
+	$($(quiet)INSTALL) -D $< $(DESTDIR)/usr/bin/$<
 
 install-care: care
 	$($(quiet)INSTALL) -D $< $(DESTDIR)/$<
 
 uninstall:
-	-$(RM) -f $(DESTDIR)/proot
+	-$(RM) -f $(DESTDIR)/usr/bin/proot
 
 uninstall-care:
 	-$(RM) -f $(DESTDIR)/care
